src/evrythng/utils.py

```diff
@@ -8,12 +8,13 @@
     from urlparse import urlparse
 
 
-# TODO: Since so much dev is going on, we've made debug=True the default. This
-# needs to change, probably soon.
 def request(request_type, resource_url, data=None, api_key=None, files=None,
-            base_url='https://api.evrythng.com', accept=False, debug=True, custom_query_params=None,
-            pageNumber=None, perPage=None, timeout=30):
+            base_url='https://api.evrythng.com', accept=False, debug=None,
+            custom_query_params=None, pageNumber=None, perPage=None, timeout=30):
     """Send a request to the Evrythng API."""
+    if debug is None:
+        debug = os.getenv('PYEVT_DEBUG', '0') == '1'
```

Comment:
why provide string variables , you can better check numeric ones, use the math luke debug = bool(os.getenv('PYEVT_DEBUG', 0))